examples/docs_snippets/docs_snippets/guides/dagster/dagster_pipes/glue/dagster_code.py

```diff
@@ -39,9 +39,6 @@ def glue_pipes_asset(
                 client=boto3.client("s3"),
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line -38]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->This predates this but does the `PipesGlueContextInjector` really provide enough value for its own class beyond `PipesS3ContextInjector`?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/23353?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
Same as with the other dummy class, the only reason I've added it was to repeat the same pattern started with Lambda Pipes. 

I was (and still am) unsure if it was necessary. We decided to go for it with @cmpadden and @smackesey in the original Glue Pipes PR. 

I think we can remove these dummy classes. They serve some "discovery" (?) purpose for a beginner Pipes user trying to connect appropriate pipes pieces together, but I don't think this role is really important. 

I'm up to removing them.